File: Lista2/C/Lsala2/Ex2.c
#include <stdio.h> 

int main()
{
	float n1, n2, media, ne, novamedia;
	printf("Nota 1: ");
	scanf("%f", &n1);
	printf("Nota 2: ");
	scanf("%f", &n2);
	media = (n1 + n2) / 2;
	if(media >= 6){
		printf("Aluno aprovado com media %.1f", media);
	}else{
		printf("Nota de exame: ");
		scanf("%f", &ne);
		novamedia = media + ne;
		if(novamedia >= 5){
			printf("Aluno aprovado em exame com media %.1f", novamedia);
		}else{
			printf("Aluno reprovado em exame com media %.1f", novamedia);
		}
	}
}

File: Lista2/Python/Lsala2/Ex2.py
n1 = float(input('Nota 1: '))
n2 = float(input('Nota 2: '))
media = (n1 + n2)/2
if media >= 6:
    print(f'Aluno aprovado com média {media}')
else:
    ne = float(input('Nota de exame: '))
    novamedia = media + ne
    if novamedia >= 5:
        print(f'Aluno aprovado em exame com média {novamedia}')
    else:
        print(f'Aluno reprovado com média {novamedia}')
